marshmallow/utils.py

```diff
@@ -210,6 +211,14 @@ def rfcformat(dt, localtime=False):
     else:
         return local_rfcformat(dt)
 
+# From Django
+_iso8601_re = re.compile(
+    r'(?P<year>\d{4})-(?P<month>\d{1,2})-(?P<day>\d{1,2})'
+    r'[T ](?P<hour>\d{1,2}):(?P<minute>\d{1,2})'
+    r'(?::(?P<second>\d{1,2})(?:\.(?P<microsecond>\d{1,6})\d{0,6})?)?'
```

Comment:
This regex was taken from Django, so I am hesitant to change it unless it is changed there as well.